src/v3/src/transformer/field/transform.ts

```diff
@@ -103,10 +155,10 @@ export const transformStepInputs = (
           validate(data) {
             // in the case of a required boolean input, just return true
             // if the backend requires the checbox val to be true, error will be displayed
-            if (typeof data[name] === 'boolean' || !!data[name]) {
```

Comment:
`!!data[name]` was originally here to check if the field was populated, and if so, short circuit and return `undefined` since we didn't have any validation for populated fields before adding max length validation